include/dts-subscription.sh

```diff
@@ -0,0 +1,336 @@
+#!/usr/bin/env bash
+
+# SPDX-FileCopyrightText: 2024 3mdeb <contact@3mdeb.com>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# shellcheck disable=SC2034
+# shellcheck source=../include/dts-environment.sh
+source $DTS_ENV
+
+check_des_creds() {
```

Comment:
@pietrushnic Should we use DPP instead of DES as in:
- https://github.com/Dasharo/docs/pull/841/files
- https://github.com/Dasharo/presentations/blob/main/dug_6_shameless_plug.md
moving forward?

Reply:
@macpijan this change was approved, and we should have everything in production using the new naming by the end of the year.

Reply:
Applied changes here 99fe04561f79370ec7245a169d18b449e8a6ba45.